src/composables/balances.ts

```diff
@@ -49,18 +50,22 @@ const initPollingWatcher = createPollingBasedOnMountedComponents();
  */
 export function useBalances({ store }: IDefaultComposableOptions) {
   const { getSdk } = useSdk({ store });
+  const { accounts, activeAccountAddress } = useAccounts({ store });
+  const { multisigAccounts } = useMultisigAccounts({ store });
 
-  const account = computed<IAccount>(() => store.getters.account);
-  const accounts = computed<IAccount[]>(() => store.getters.accounts);
   const currentCurrencyRate = computed(() => store.getters.currentCurrencyRate);
 
-  const balance = computed(() => balances.value[account.value.address] || new BigNumber(0));
+  const balance = computed(() => balances.value[activeAccountAddress.value] || new BigNumber(0));
   const balanceCurrency = computed(() => balance.value.toNumber() * currentCurrencyRate.value);
 
   async function updateBalances() {
     const sdk = await getSdk();
-    const balancesPromises = accounts.value.map(
-      ({ address }) => sdk.balance(address).catch((error) => {
+    const addresses: string[] = [
+      ...accounts.value.map((acc) => acc.address),
+      ...multisigAccounts.value.map((acc) => acc.address),
```

Comment:
I strongly suggest to decouple multisig feature from the regular accounts. If you want to have the balances polling I suggest to create separate composable `multisigBalances`. Why? because if user is switched to the regular accounts view there is no need to poll the multisig balances and vice versa.

Also there is no such a thing as multisig address. It is `gaAccountId`.